src/DB_Command.php

```diff
@@ -474,7 +474,15 @@ public function export( $args, $assoc_args ) {
 			$assoc_args['result-file'] = $result_file;
 		}
 
-		$command = '/usr/bin/env mysqldump --no-defaults %s';
+		$mysqldump_version = exec( "/usr/bin/env mysqldump --version --no-defaults" );
+		$mysqldump_version = preg_replace( "/^.+([0-9]+\.[0-9]+\.[0-9]+)(.+)$/", "$1", $mysqldump_version );
```

Comment:
This test does not take MariaDB into account.

Here's a version string from MariaDB from my current local setup:
```
$ /usr/bin/env mysqldump --version --no-defaults
mysqldump  Ver 10.16 Distrib 10.2.14-MariaDB, for osx10.13 (x86_64)
```

So you not only need to check the version, but also whether you actually are using MySQL, not some variant.